src/scripts/webpack/config.js

```diff
@@ -48,19 +48,22 @@ const getConfigType = (configFolderPath) => {
 };
 
 const nodeEnv = process.env.NODE_ENV || 'development';
-const envWhitelist = ['apiEndpoint', 'fetchLocalesFrom'];
```

Comment:
uhm, i think we might have a problem here:

```
npm start

  scriptoni:webpack building with NODE_ENV=development +0ms
  scriptoni:webpack Configuration {
    "port": 9091,
    "devTool": "source-map",
    "title": "DEV: Alinity PRO",
    "bundle": {
        "apiEndpoint": "https://corsproxy.our.buildo.io/http://alinity-pro-dev.omnilab-development.com:8008/api",
        "username": "gonzo",
        "password": "giogio",
        "debug": "scriptoni:*"
    }
} +1ms
```

`gonzo` is my username, not the one provided in `local.json`

Moreover, this doesn't work either:

```
USERNAME=jonny npm start

  scriptoni:webpack building with NODE_ENV=development +0ms
  scriptoni:webpack Configuration {
    "port": 9091,
    "devTool": "source-map",
    "title": "DEV: Alinity PRO",
    "bundle": {
        "apiEndpoint": "https://corsproxy.our.buildo.io/http://alinity-pro-dev.omnilab-development.com:8008/api",
        "username": "gonzo",
        "password": "giogio",
        "debug": "scriptoni:*"
    }
} +1ms

```